File: Example 4,5,6.js
const address = 'DKI Jakarta';
const addresses = new Array(10).fill(address)

function findAddress(addresses) {
	 const tx = performance.now();
	 console.log('The Default Address is ' + (addresses[0]));
	 const ty = performance.now();
	 console.log('The Performance is ' + (ty-tx) + ' ms');
}
findAddress(addresses);

File: Example 4,5,6.go
package main

import (
	"fmt"
	"time"
)

func findAddress(addresses []string) {
	var tx int64 = 0 // inisialisasi tx dengan nilai 0
	ty := time.Now().UnixNano()
	fmt.Println("The Default Address is", addresses[0])
	fmt.Println("The Performance is", (ty-tx)/1000000, "ms")
}

func main() {
	address := "DKI Jakarta"
	addresses := make([]string, 10)
	for i := range addresses {
		addresses[i] = address
	}
	tx := time.Now().UnixNano()
	findAddress(addresses)
	ty := time.Now().UnixNano()
	fmt.Println("The Performance is", (ty-tx)/1000000, "ms") // menghitung waktu akhir di dalam fungsi main()
}